jenkins/manila-functional-tests/fabfile.py

```diff
@@ -0,0 +1,432 @@
+import datetime
+import fabric
+import io
+import json
+import os
+import time
+
+import heatclient.client
+import keystoneclient.auth.identity.v2
+import keystoneclient.session
+
+from fabric.api import env, execute, task, put, roles, run, parallel, sudo
+from fabric.context_managers import cd, hide, prefix, settings, shell_env
+from fabric.contrib.files import append, contains, sed, upload_template
+from heatclient.common import template_utils
+
+
+@roles('nfs_connector', 'cifs_connector')
+@parallel
+def add_apt_repositories(credentials):
+    """
+    Add Scality APT repositories.
+
+    :param credentials: credentials for packages.scality.com
+    :type credentials: string
+    """
+    release = 'stable_lorien'
+    gpg_key = '4A23AD0E'
+    repository = (
+        'http://{auth:s}@packages.scality.com/{release:s}/ubuntu'.format(
+            auth=credentials,
+            release=release,
+        )
+    )
+
+    # Add GPG key.
+    put('../scality5.gpg', '/tmp')
```

Comment:
It would be best to have this gpg key in that repo so that we don't depend on the file hierarchy and file names in the openstack-ci-repo.